最新主分支编译报错

**Question**
In file included from /opt/work/src/miniob/src/oblsm/compaction/ob_compaction.h:13,
                 from /opt/work/src/miniob/src/oblsm/compaction/ob_compaction.cpp:11:
/opt/work/src/miniob/src/oblsm/table/ob_sstable.h:140:63: error: ‘maybe_unused’ attribute ignored [-Werror=attributes]
  140 |   [[maybe_unused]] ObLRUCache<uint64_t, shared_ptr<ObBlock>> *block_cache_;
      |                                                               ^~~~~~~~~~~~
cc1plus: all warnings being treated as errors
make[2]: *** [src/oblsm/CMakeFiles/oblsm.dir/build.make:76: src/oblsm/CMakeFiles/oblsm.dir/compaction/ob_compaction.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:479: src/oblsm/CMakeFiles/oblsm.dir/all] Error 2

g++ 11.2.1
cmake 3.24.4
flex 2.5.37
bison 3.8

--- a/src/oblsm/table/ob_sstable.h
+++ b/src/oblsm/table/ob_sstable.h
@@ -72,7 +72,9 @@ public:
         comparator_(comparator),
         file_reader_(nullptr),
         block_cache_(block_cache)
-  {}
+  {
+    (void)block_cache_;
+  }
 
   ~ObSSTable() = default;
 
@@ -137,7 +139,7 @@ private:
   unique_ptr<ObFileReader> file_reader_;
   vector<BlockMeta>        block_metas_;
 
-  [[maybe_unused]] ObLRUCache<uint64_t, shared_ptr<ObBlock>> *block_cache_;
+  ObLRUCache<uint64_t, shared_ptr<ObBlock>> *block_cache_;
 };
 
 class TableIterator : public ObLsmIterator

